Documentation/ceph-examples.md

```diff
@@ -84,4 +84,4 @@ Object storage contains multiple pools that can be configured for different scen
 See the [Object Store CRD](ceph-object-store-crd.md) topic for more details on the settings.
 
 ### Object Storage User
-- `object-user.yaml`: Creates a simple object storage user and generates creds for the S3 API
+- `object-user.yaml`: Creates a simple object storage user and generates credentials for the S3 API.
```

Comment:
The trend I see in most of the rest of the docs and the guideline I see most often is to make sure bulleted list items do *not* end in a period.